deploy/cluster-monitoring-config-non-uwm/config.yaml

```diff
@@ -3,8 +3,8 @@ selectorSyncSet:
   matchLabelsApplyMode: "OR"
   matchExpressions:
     - key: hive.openshift.io/version-major-minor
-      operator: NotIn
-      values: ["4.6", "4.7", "4.8"]
+      operator: In
+      values: ["4.5"]
```

Comment:
how many 4.5s do we have left? This specific condition can hopefully go away soon